src/modules/redesign/components/assets/einsatz.vue

```diff
@@ -32,9 +30,7 @@ export default Vue.extend<
     name: 'lssmv4-redesign-einsatz-component',
     computed: {
         mission() {
-            return useAPIStore().missionsArray.find(
-                ({ id }) => id === this.type.toString()
-            );
+            return undefined;
```

Comment:
Do Vue need a return value? This seems weird.

Reply:
no, this is more a way to not have issues with async operations. This is more or less okayish as this component is not yet in use.